C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.11_3.11.2544.0_x64__qbz5n2kfra8p0\Lib\pickle.py:1718: UserWarning: [01:35:10] WARNING: C:\actions-runner\_work\xgboost\xgboost\src\data\../common/error_msg.h:82: If you are loading a serialized model (like pickle in Python, RDS in R) or
configuration generated by an older version of XGBoost, please export the model by calling
`Booster.save_model` from that version first, then load it back in current version. See:

    https://xgboost.readthedocs.io/en/stable/tutorials/saving_model.html

for more details about differences between saving model and serializing.

  setstate(state)
Exception in Tkinter callback
Traceback (most recent call last):
  File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.11_3.11.2544.0_x64__qbz5n2kfra8p0\Lib\tkinter\__init__.py", line 1967, in __call__
    return self.func(*args)
           ^^^^^^^^^^^^^^^^
  File "G:\ai projects\AiUserFriendlyProject - DbConnected\app.py", line 116, in <lambda>
    command=lambda: predict_strength(),
                    ^^^^^^^^^^^^^^^^^^
  File "G:\ai projects\AiUserFriendlyProject - DbConnected\app.py", line 228, in predict_strength
    threading.Thread(target=run_prediction).start()
    ^^^^^^^^^
NameError: name 'threading' is not defined
